comps/comps-foreman-plugins-rhel7.xml

```diff
@@ -82,6 +82,7 @@
       <packagereq type="default">tfm-rubygem-foreman_setup</packagereq>
       <packagereq type="default">tfm-rubygem-foreman_snapshot_management</packagereq>
       <packagereq type="default">tfm-rubygem-foreman_spacewalk</packagereq>
+      <packagereq type="default">tfm-rubygem-foreman_statistics</packagereq>
```

Comment:
can you please add similar changes (minus the `tfm-` prefix) to `comps-foreman-plugins-el8.xml` please? You can use `add_to_comps.rb` for that.

Reply:
sure, would be nice to add it to the `add_gem_package.sh` script as well :thinking: :)

Reply:
yeah. it's a bit broken for the two-os usecase we have right now.